at_secondary/at_secondary_server/lib/src/server/at_secondary_impl.dart

```diff
@@ -310,10 +310,10 @@ class AtSecondaryServerImpl implements AtSecondaryServer {
     try {
       command = SecondaryUtil.convertCommand(command);
       await executor!.execute(command, connection, verbManager!);
-    } on Exception catch (e, trace) {
+    } on Exception catch (e) {
       await GlobalExceptionHandler.getInstance()
           .handle(e, atConnection: connection);
-    } on Error catch (e, trace) {
+    } on Error catch (e) {
       logger.severe(e.toString());
```

Comment:
Just thinking. Does adding some additional context to the logs like 'Exception in executing verb etc...'

Reply:
Sure, will add the logs.